springdoc-openapi:generate Connection refused: connect

# Reproduktion

- dev-branch auschecken
- im Briefwahl-Service `mvn clean verify` ausführen
- es kommt beim `springdoc-openapi:1.4:generate` die Fehlermeldung  
```
[INFO] --- springdoc-openapi:1.4:generate (default) @ wls-briefwahl-service ---
[ERROR] An error has occured
java.net.ConnectException: Connection refused: connect
    at sun.nio.ch.Net.connect0 (Native Method)
    at sun.nio.ch.Net.connect (Net.java:589)
    at sun.nio.ch.Net.connect (Net.java:578)
```
- ein `openapi.json` ist nicht vorhanden

# Ziel
Das File `openapi.json` sollte erstellt werden (#320)

--- a/wls-briefwahl-service/src/main/resources/application.yml
+++ b/wls-briefwahl-service/src/main/resources/application.yml
@@ -31,7 +31,7 @@ realm: wls_realm
 
 server:
   shutdown: "graceful"
-  port: 39146
+  port: 8080
   error:
     include-exception: false
     include-stacktrace: never

